### PR TITLE
FLOW-6442- new card template for RBM

### PR DESCRIPTION
**Pull request by wreakhead**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -853,6 +899,14 @@ Component that represents a URL to an image, video or audio file. Used within ge
 ### new Media()
 
 
+| Param | Type | Description |
+| --- | --- | --- |
+| opts.url | <code>string</code> | Required |
+| opts.type | <code>string</code> | Required |
```
Comment (eeshanjain) [inline README.md:905]: Is this really needed? It seems like it is being repeated, if I look at the README. Please have a look at the screenshot below.
image

File: src/rbm/components/media.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    if(typeof type !== 'string' || type.length === 0) {
+      throw new Error(`type is mandatory for a Media component`)
+    }
+    if(typeof height !== 'string' || height.length === 0) {
+      throw new Error(`height is mandatory for a Media component`)
+    }
+    if (type === 'video' && thumbnailUrl && typeof thumbnailUrl !== 'string') {
+      throw new Error('Thumbnail should be of type string')
```
Comment (eeshanjain) [inline src/rbm/components/media.js]: Please change the exception to `'Thumbnail should be of type string for a video'`. This adds some more context.

File: src/rbm/components/media.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    if(type === 'video' && thumbnailUrl && thumbnailUrl.length && !/^https?:\/\//.test(thumbnailUrl)) {
+      throw new Error('Thumbnail should be valid url')
```
Comment (eeshanjain) [inline src/rbm/components/media.js]: Similar to above comment, we can modify the exception to `'Video thumbnail should be valid url'`

File: README.md (hunk 1 above)
Comment (wreakhead) [reply README.md:905]: it is auto generated, fixed in latest commit

File: src/rbm/components/media.js (hunk 2 above)
Comment (wreakhead) [reply src/rbm/components/media.js]: fixed in latest commit

File: src/rbm/components/media.js (hunk 3 above)
Comment (wreakhead) [reply src/rbm/components/media.js]: fixed in latest commit

**Pull request merged**